packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -77,6 +77,9 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     throw new TRPCError({ code: "UNAUTHORIZED" });
   }
 
+  if (!Number(rest.length))
+    throw new TRPCError({ code: "BAD_REQUEST", message: "The duration must be at least 1 minute." });
+
```

Comment:
i think we can check this in input schema.

in packages/prisma/zod/eventtype.ts we can add .min(1) to length

Reply:
As the zod schema files are generated by the zod-prisma and are not tracked by git, shall I add the statement in the prisma.schema?
![image](https://github.com/calcom/cal.com/assets/86384652/9fb541c3-be61-4507-9942-ec1ac33e6a22)

Reply:
no actually i meant to add min() in the zod input schema not the schema.prisma.  in the file packages/prisma/zod/eventtype.ts

Reply:
![image](https://github.com/calcom/cal.com/assets/86384652/59a7e59b-c56a-40f7-8d7a-e8b89ae4a768)

You are talking about here right? 
This file is not tracked by git and is automatically generated by the prisma-zod on db migration. So it will be over-written by the schema defined in schema.prisma.

Reply:
oh okay i think you would have to create a migration

Reply:
yarn workspace @calcom/prisma db-migrate

Reply:
As there are no changes to the database schema itself I dont think creating a migration will be necessary or even possible.
  
```sh
yarn workspace @calcom/prisma generate-schemas
```
```sh
yarn workspace @calcom/prisma db-migrate
```
We can do either of these as commands as the _db-migrate_ will automatically run the _generate-schemas_ command.

Reply:
yarn workspace @calcom/prisma db-migrate this commands also generates zod schema

<img width="721" alt="Screenshot 2023-09-05 at 5 59 41 PM" src="https://github.com/calcom/cal.com/assets/53316345/dab90fec-041d-467c-a3e6-e99ec70425ba">